client/src/components/App.jsx

```diff
@@ -26,10 +27,22 @@ class App extends React.Component {
     .catch(() => {console.log("Error: Reviews not found.")}
   )};
 
+  renderReviews() {
+    return (
+      <Reviews sampleReviews={this.state.reviews}/>
```

Comment:
"sampleReviews" can be the value of your this.state.reviews, but I'd pass it into the components with a different variable name (just "reviews").
this is important when you're app is fully functional, when you're component is no longer using sample data.